sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -827,6 +829,12 @@ def dynamic_queries(self):
             else:
                 self.log_missing_metric("Failover Cluster Instance", major_version, engine_edition)
 
+        if is_affirmative(self.instance.get('include_log_shipping_primary', False)):
+            queries.extend([LOG_SHIPPING_PRIMARY])
+
+        if is_affirmative(self.instance.get('include_log_shipping_secondary', False)):
+            queries.extend([LOG_SHIPPING_SECONDARY])
```

Comment:
The config mentioned to run against a instance that is in `Transaction Log Shipping configuration`. Do we want to systematically check that before executing the query?

Reply:
I don't think it's necessary, since the query will just return an empty set if there's no log shipping going on. The more important thing is that it requires the user to configure something custom for the `datadog` user. If we merge this PR i'll need to update the setup docs.